lib/bcu.rb

```diff
@@ -82,8 +130,9 @@ def self.find_outdated_apps(quiet)
 
     installed.each do |app|
       version_latest = (app[:version] == "latest")
-
-      if options.force && options.all && version_latest && app[:auto_updates]
+      if pins.include?(app[:token])
+        state_info[app] = "pinned"
```

Comment:
For some reason doesn't work properly under certain conditions.
I pinned 3 casks as following:
```bash
> brew cu --pins                                                                                                                    
toggl
gimp
fork
```

And when I run `brew cu --all --cleanup` this is what I get:
```bash
> brew cu --all --cleanup                                                                                                           
==> Options
Include auto-update (-a): true
Include latest (-f): false
==> Updating Homebrew
Already up-to-date.
==> Finding outdated apps
       Cask     Current     Latest     A/U    Result
...
21/62  fork     1.0.72.2    1.0.72.2    Y   [   OK   ]
22/62  gimp     2.10.8,2    2.10.8,2        [ PINNED ]
...
53/62  toggl    7.4.330     7.4.331         [ PINNED ]
...
``` 

My guess is that the `auto-update` flag is somehow interfering with the pinning implementation.

Reply:
Looks good now 👍